package/rmt-server.spec

```diff
@@ -40,6 +40,14 @@ Source0:        %{name}-%{version}.tar.bz2
 Source1:        rmt-server-rpmlintrc
```

Comment:
We should probably bump the version for this as well

Reply:
version has been bumped to "0.0.3"